docs/security.rst

```diff
@@ -104,3 +104,127 @@ vulnerabilities
 <https://github.com/pallets/flask/issues/248#issuecomment-59934857>`_, so
 this behavior was changed and :func:`~flask.jsonify` now supports serializing
 arrays.
+
+
+SSL/HTTPS
+---------
+
+For implementing HTTPS on your server.
+
+Below are some packages that implement this protocol:
+
+* `flask-talisman <https://github.com/GoogleCloudPlatform/flask-talisman>`_
+* `flask-sslify <https://github.com/kennethreitz/flask-sslify>`_
+* `flask-secure-headers <https://github.com/twaldear/flask-secure-headers>`_
+
+Security Headers
+----------------
+
+This section contains a list of headers supported by Flask and some packages that implements them.
+
+Content Security Policy (CSP)
+-----------------------------------------------------------------------------
+
+Enhance security and prevents common web vulnerabilities such as cross-site scripting and MITM related attacks.
+
+Example:
+
+.. sourcecode:: html
+   
+   Content-Security-Policy: default-src https:; script-src 'nonce-{random}'; object-src 'none'
+
+
+See also `Content Security Policy <https://csp.withgoogle.com/docs/index.html>`_.
+
+* `flask-talisman <https://github.com/GoogleCloudPlatform/flask-talisman>`_
+* `flask-csp <https://github.com/twaldear/flask-csp>`_
+* `flask-secure-headers <https://github.com/twaldear/flask-secure-headers>`_
+
+HTTP Strict Transport Security (HSTS)
+------------------------------------------------------------------------------------------------------------------------------
+
+Redirects http requests to https on all urls, preventing MITM attacks.
+
+Example:
+
+.. sourcecode:: html
```

Comment:
it's not really html. is there a better syntax (otherwise, what about don't specifying one at all) that highlights http headers nicely?